Tests/SourceKittenFrameworkTests/VariantPerformanceTests.swift

```diff
@@ -128,6 +128,26 @@ class VariantPerformanceTests: XCTestCase {
         }
         XCTAssertEqual(availables.sorted(), self.expectedAvailables)
     }
+
+    func testFindAvailablesWithVariantUIDNamespace() {
+        func findAvailables(variant: SourceKitVariant) -> [String] {
+            let resultFromSubstructure = variant.subStructure?.flatMap(findAvailables) ?? []
+            if variant.kind == UID.source.lang.swift.decl.function.method.instance,
+                let attributes = variant.attributes?.flatMap({ $0.attribute }),
+                attributes.contains(.available),
```

Comment:
Now, some properties of `SourceKitVariant` returns specific types of `UIDNapespace`.
e.g. `attribute` returns `UID.source.decl.attribute`, so it can be equatable with `.available` instead of `UID.source.decl.attribute.available`.